playbooks/roles/monitoring-user-defined-projects/tasks/main.yml

```diff
@@ -0,0 +1,169 @@
+---
+
+## Enabling monitoring to user-defined-projects
+
+- set_fact:
+    default_namespace: "{{ '-'.join([query('community.general.random_string', upper=false, numbers=false, special=false)[0],'namespace']) }}"
```

Comment:
This requires Community General Collection which requires additional steps, can we do away with that and use a random number instead?

Reply:
Implemented the random number instead of random string module in community general collection.